guides/common/modules/ref_log-file-directories.adoc

```diff
@@ -1,23 +1,22 @@
 [id="Log_File_Directories_{context}"]
 = Log File Directories Provided by {Project}
```

Comment:
```suggestion
= Log File Directories
```

Alternative: "Log File Directories on {ProjectServer}". Your call; but please ensure file name, anchor, and title matches.

Reply:
Applied.